mobify/platform-scaffold
# [WEB-1423]  add to wishlist from PDP

This PR adds an add to wishlist button to the PDP. 

 **JIRA**: https://mobify.atlassian.net/browse/WEB-1423
 **Linked PRs**: https://github.com/mobify/progressive-web-sdk/pull/754

## Changes
- Add wishlist button to the PDP
- Add action to PDP that calls integration manager add to wishlist command
- Moved addToWishlist command in merlins, SFCC and stub connectors to products branch (and rename) 
- Set sign in and wishlist URLs 
- Removed deprecated registerConnectorExtension

## Todos

## How to test-drive this PR
- checkout the move-add-to-wishlist-command branch of the SDK
- run `npm run dev:build`
- run `npm link`
- cd into the scaffold web folder
- run `npm link progressive-web-sdk`
- preview the merlin's site
- navigate to a PDP
- try adding the item to your wishlist
- you should be taken to the login page
- sign into an account
- navigate back to a PDP
- try adding the item to your wishlist
- you should see a modal stating the item was added to your wishlist
- repeat previewing the SFCC site
